mu-plugins/blocks/global-header-footer/footer.php

```diff
@@ -98,7 +98,7 @@
 	<!-- wp:social-links {"className":"is-style-logos-only"} -->
 	<ul class="wp-block-social-links is-style-logos-only">
 		<!-- wp:social-link {"url":"https://www.facebook.com/WordPress/","service":"facebook","label":"<?php echo esc_html_x( 'Visit our Facebook page', 'Menu item title', 'wporg' ); ?>"} /-->
-		<!-- wp:social-link {"url":"https://twitter.com/WordPress","service":"twitter","label":"<?php echo esc_html_x( 'Visit our Twitter account', 'Menu item title', 'wporg' ); ?>"} /-->
+		<!-- wp:social-link {"url":"https://www.x.com/WordPress","service":"x","label":"<?php echo esc_html_x( 'Visit our X account', 'Menu item title', 'wporg' ); ?>"} /-->
```

Comment:
Should we add (formerly Twitter)? They've taken that approach in the App store.

<img width="628" alt="Screenshot 2023-09-27 at 2 37 25 PM" src="https://github.com/WordPress/wporg-mu-plugins/assets/1657336/e1f98de9-763f-4941-8cd2-063b45e3bed9">

Reply:
Good call. Just updated the PR.